README.md

```diff
@@ -0,0 +1,42 @@
+
+# Hackers Conference 2022 
+
+## this is a webpage for a conference
+
+## Built With
+- html
+- css
+- javascript 
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+## Step 1:
+- if you don't have git installed on your machine you can download it from [here ](https://git-scm.com/downloads).
+- Once you have git installed on your machine you can clone your project by running the command below to clone this project to your local machine in the directory of the folder you want it to be cloned
+- git clone https://github.com/quavo19/FristCapStone.git
+- Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
+
+## Step 2: 
+- Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
+
+
+## Author
+👤 Donald Akite
+
+- GitHub: @ https://github.com/quavo19
+- Twitter:@ https://twitter.com/DonaldAkite
+
+## 🤝 Contributing
+- Contributions, issues, and feature requests are welcome!
+
+## Live Demo 
+
+you can view a live demo from [here](https://quavo19.github.io/FristCapStone/)
+
```

Comment:
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).